src/main/java/lotto/model/LottoHandler.java

```diff
@@ -0,0 +1,104 @@
+package lotto.model;
+
+import lotto.constants.Ranking;
+import lotto.constants.lottoType.LottoType;
+import lotto.constants.messageType.OutputMessageType;
+import lotto.utils.WinningNumberValidation;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LottoHandler {
+
+    private List<Lottos> lottos = new ArrayList<>();
+    private Lotto winningLottoNumbers;
+    private int bonusNumber;
+
+    public void buyLottos(int lottoTickets) {
+        for (int num = 0; num < lottoTickets; num++) {
+            Lottos lottoNumbers = Lottos.generateLottoNumbers();
+            lottos.add(lottoNumbers);
+        }
+    }
+
+    public String getAllLottos() {
+        return getLottos().stream()
```

Comment:
```suggestion
        return lottos.stream()
```
이 처럼 getter 를 사용하지 않고 바로 사용할 수 있을 것 같아요!

이 외에도 winningLottoNumbers 와 같이 필드 변수를 메서드의 인자로 사용하고 있는데, 그렇게 설계하신 이유가 궁금해요!
가령 checkedResult() 메서드가 Lotto 를 사용하는 이유가 오직 테스트하기 위함이라면, 필드변수로 Lotto 객체가 있는 설계와 맞지 않는 것 같다고 느껴져요.
이에 대해 고민해보시면 좋겠어요!